tracer/src/Datadog.Trace/AppSec/Waf/Encoder.cs

```diff
@@ -18,9 +18,28 @@ namespace Datadog.Trace.AppSec.Waf
 {
     internal static class Encoder
     {
+        private const int MaxBytesForMaxStringLength = WafConstants.MaxStringLength * 4;
```

Comment:
is it because each char is worth 4 bytes?

Reply:
this is because the max bytes for an utf8 char is 4 bytes yes.

Reply:
ok thanks!